boards/limifrog-v1/include/periph_conf.h

```diff
@@ -139,37 +139,27 @@ static const timer_conf_t timer_config[] = {
 #define I2C_IRQ_PRIO        1
 #define I2C_APBCLK          (36000000U)
 
+static const i2c_conf_t i2c_config[] = {
+    /* device, port, scl-, sda-pin-number, scl-, sda-AF, ER-IRQn, EV-IRQn */
+    {I2C1, GPIOB,  8,  9, 4, 4, I2C1_ER_IRQn, I2C1_EV_IRQn},
+    {I2C2, GPIOB, 10, 11, 4, 4, I2C2_ER_IRQn, I2C2_EV_IRQn},
+};
+
```

Comment:
AFAIK the i2c on the stm32 can work either interrupt based or with busy waits, which is how it's currently implemented. The event interrupt handling is not used by any of the stm32 in RIOT :+1: